package/firmware/ipq-wifi/Makefile

```diff
@@ -27,6 +27,7 @@ endef
 ALLWIFIBOARDS:= \
 	8dev_habanero-dvk \
 	aruba_ap-303 \
+	asus_rt-acrh17 \
```

Comment:
Did you sent the boardfiles for both wifis upstream?

Reply:
TODO, I'll leave this comment open

Reply:
Ok, from what I can see, everything else looks ok. I'll wait for the posts to show up on the ML. Note: It's enough to post them. What they do with it is upstreams own choice.

Reply:
https://lists.infradead.org/pipermail/ath10k/2022-January/013217.html and https://lists.infradead.org/pipermail/ath10k/2022-January/013218.html